README.rst

```diff
@@ -31,10 +31,18 @@ Server example
 
     from aiohttp import web
     from aiohttp_xmlrpc import handler
-    from tornado.testing import *
 
 
     class XMLRPCExample(handler.XMLRPCView):
+        # Specify 'METHOD_NAMES' only for the ones with
+        # a different name than the standard one (name without "rpc_").
+        #
+        # Format: { "name_to_use": "method_name_in_class"}
+        METHOD_NAMES = {
+            "nested.test": "rpc_test",
```

Comment:
Could you want to reverse this map? That's will be better on lookup and didn't allow make replacement twice.

Reply:
No need to change that since we are not using the "METHOD_NAMES" dict anymore.